S02-literals/quoting.t

```diff
@@ -524,6 +527,8 @@ Hello, World
     my $var = 'world';
     is  qx/echo world/.chomp, "world", 'qx';
     is qqx/echo $var/.chomp,  "world", 'qqx';
+    is Qx[echo '\\'].chomp, "\\\\", "Qx treats backslash literally";
```

Comment:
This test fails. With different failure modes on Debian and on Windows:

**Debian:**
```
not ok 1 - Qx treats backslash literally

# Failed test 'Qx treats backslash literally'
# at z.p6 line 4
# expected: '\\'
#      got: '\'
```

**Windows:**
```
not ok 1 - Qx treats backslash literally

# Failed test 'Qx treats backslash literally'
# at p.t line 4
# expected: '\\'
#      got: ''\\''
```